### PR TITLE
don't automatically run translations updates, add a manual target

### PR DESCRIPTION
**Pull request by mzanetti**

### CODE REVIEW

**Comment by mzanetti** [comment]
Jenkins, land this please

**Comment by mzanetti** [comment]
Jenkins, land this please

**Comment by mzanetti** [comment]
silo this

**Comment by mzanetti** [comment]
silo this

**Comment by mzanetti** [comment]
Jenkins, retest this please

**Comment by mzanetti** [comment]
silo this

**Comment by mzanetti** [comment]
silo this

**Comment by mzanetti** [comment]
silo this

File: .gitignore [hunk 1]
```diff
@@ -3,7 +3,6 @@
 builddir
 doc/html
 *api.json.new
-*.qm
```
Comment (t-mon) [inline .gitignore]: we should never ship qm file with the source since they get generated at build time

Comment (mzanetti) [reply .gitignore]: yep, which is why I removed that from the gitignore, so that those files can be delete with git clean-tree

**Comment by mzanetti** [comment]
Jenkins, land this please

**Comment by mzanetti** [comment]
Jenkins, land this please

**Pull request merged**